testData/highlighting/equalinif.go

```diff
@@ -0,0 +1,29 @@
+package equalinif
+
+func <warning descr="Unused function 'eqlfunc'">eqlfunc</warning>() {
```

Comment:
Ah, I thought I need to do something like `<warning descr="_">eqlfunc</warning>`. Done this as well. Thank you.